test-suite/misc/coqc_dash_vok.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+IN_V=misc/coqc_cmdline.v
+OUT_VO=misc/coqc_cmdline.vo
+OUT_VIO=misc/coqc_cmdline.vio
+OUT_VOS=misc/coqc_cmdline.vos
+OUT_VOK=misc/coqc_cmdline.vok
+OUT_GLOB=misc/coqc_cmdline.glob
+OUT="${OUT_VO} ${OUT_VIO} ${OUT_VOS} ${OUT_VOK} ${OUT_GLOB}"
+
+rm -f ${OUT}
+
+set -x
+
+$coqc ${IN_V} -vos
+$coqc ${IN_V} -vok
+if [ ! -f ${OUT_VOK} ]; then
+  printf "coqc -vok not working in -vos mode"
```

Comment:
```suggestion
  echo "coqc -vok not working in -vos mode"
```
or 
```suggestion
  printf 'coqc -vok not working in -vos mode\n'
```
(I like single quotes for printf better but the important part is the \n)